### PR TITLE
po/usermanual

### PR DESCRIPTION
**Pull request by TurboGit**
Native English speaker review welcomed !

### CODE REVIEW

File: doc/usermanual/preferences/darkroom_options.xml [hunk 1]
```diff
@@ -101,11 +101,30 @@
...
-    
+
     <bridgehead renderas="sect5">reduce resolution of preview image</bridgehead>
 
     <para>
-      Decrease to speed up preview rendering, may hinder accurate masking.
+      This reduces the resolution of the preview window at the
+      upper-left of the screen. Consequently, calculations for
+      re-drawing this window are reduced to 1/4, 1/9 or 1/16 of the
+      main window. Sometimes coordinates for handles are borrowed from
```
Comment (pitbuster) [inline doc/usermanual/preferences/darkroom_options.xml:111]: I would phrase it as: "Full scale may be needed for precise work on color pickers and/or masks."

File: doc/usermanual/darkroom/modules/color/colorbalance.xml [hunk 2]
```diff
@@ -65,6 +65,10 @@
 
       </itemizedlist>
 
+      By clicking on the shadows label different layouts of the controls
```
Comment (pitbuster) [inline doc/usermanual/darkroom/modules/color/colorbalance.xml:68]: I propose a phrasing like this:
< By clicking on the shadows label different layouts of the controls are proposed. 
> Three different control layouts can be accessed by clicking on the shadows label:

**Review by pitbuster — CHANGES_REQUESTED** [review]
I am no native (but I write a lot of English for my work) and I proposed a bit simpler phrasing in some parts. The rest of the updates looks good :)

File: doc/usermanual/preferences/darkroom_options.xml (hunk 1 above)
Comment (TurboGit) [reply doc/usermanual/preferences/darkroom_options.xml:111]: @pitbuster : this chunk has been written by a native English, so won't edit it :)

Comment (pitbuster) [reply doc/usermanual/preferences/darkroom_options.xml:111]: At least the combination of "sometimes" and "rarely" in the same phrase in the same sentence is weird for a manual, but it is your call.

**Review by pitbuster — APPROVED** [review]

**Pull request merged**